mne/beamformer/_rap_music.py

```diff
@@ -212,7 +182,7 @@ def _compute_proj(A):
 
 @verbose
 def rap_music(
-    evoked, forward, noise_cov, n_dipoles=5, return_residual=False, verbose=None
+    evoked, forward, noise_cov, n_dipoles=5, return_residual=False, verbose=None, use_trap=False
```

Comment:
By convention `verbose` is always last, and we should add `*` to prevent errors at the user end
```suggestion
    evoked, forward, noise_cov, n_dipoles=5, return_residual=False, *, use_trap=False, verbose=None
```

Reply:
Done.